src/query/xasl_cache.c

```diff
@@ -2223,9 +2223,10 @@ xcache_cleanup (THREAD_ENTRY * thread_p)
       while ((xcache_entry = iter.iterate ()) != NULL && count < xcache_Soft_capacity)
 	{
 	  candidate.xid = xcache_entry->xasl_id;
-	  candidate.xcache = xcache_entry;
-	  if (candidate.xid.cache_flag > 0 || (candidate.xid.cache_flag & XCACHE_ENTRY_FLAGS_MASK)
```

Comment:
candidate.xid.cache_flag > 0 means !XCACHE_ENTRY_MARK_DELETED and XCACHE_ENTRY_FIX_COUNT_MASK > 0.
can it be deleted?

Reply:
I reverted to 10.2. The original code of 10.2 did not check cache_flag, so I modified the code but I reverted it because it might affect performance (ex. YCSB). I think this code does not affect the performance and I will revert again.